packages/textfield/src/textfield.ts

```diff
@@ -76,6 +76,9 @@ export class Textfield extends Focusable {
     @property({ type: Boolean, reflect: true })
     public required = false;
 
+    @property({ type: String, reflect: true })
+    public autocomplete?: string;
```

Comment:
For future proofing you could probably type this as:

```
public autocomplete?: HTMLInputElement['autocomplete'] | HTMLTextAreaElement['autocomplete']
```

It appears TS's `dom.d.ts` currently actually has autocomplete as a string, its just `lit-analyzer` which is getting its types from the IDL somehow. I would hope sometime in the future TS provides better types for these.

Reply:
@benjamind Ok, I am trying out your version. That is a clever trick.